e2etests/testcase_test.go

```diff
@@ -2768,6 +2768,34 @@ var testCases = []testCase{
 				AddedBy: "sha256:2408cc74d12b6cd092bb8b516ba7d5e290f485d3eb9672efc00f0583730179e8",
 				FixedBy: "1.35.0-r15",
 			},
+			{
+				Name:          "openssl",
+				NamespaceName: "alpine:v3.16",
+				VersionFormat: "apk",
+				Version:       "1.1.1a-r0",
+				Vulnerabilities: []apiV1.Vulnerability{
+					{
+						Name:          "CVE-2018-0734",
```

Comment:
This vuln is fixed by `1.1.1a`. I'd expect us to find `CVE-2022-2097`, which is fixed by `1.1.1q-r0`